test/e2e/performanceprofile/functests/0_config/config.go

```diff
@@ -158,9 +159,40 @@ func externalPerformanceProfile(performanceManifest string) (*performancev2.Perf
 	return profile, nil
 }
 
-func testProfile() *performancev2.PerformanceProfile {
+func testProfile() (*performancev2.PerformanceProfile, error) {
 	reserved := performancev2.CPUSet("0")
 	isolated := performancev2.CPUSet("1-3")
+	profileCpus := &performancev2.CPU{
+		Reserved: &reserved,
+		Isolated: &isolated,
+	}
+
+	customReserved, foundReserved := os.LookupEnv("RESERVED_CPU_SET")
+	customIsolated, foundIsolated := os.LookupEnv("ISOLATED_CPU_SET")
+	if foundReserved != foundIsolated {
+		return nil, fmt.Errorf("both reserved and isolated cpusets are required.")
```

Comment:
we need to either provide both or none - let's reflect this in the error message